### DISCUSSION TITLE
[BUG] Gui doesn't load fully

### DISCUSSION THREAD

**Discussion by securemeit** [utmstack/UTMStack#726] [Q&A]
**Describe the bug**

Gui doesn't load fully after upgrade to version 10.5.1-202406182313

**To Reproduce**

After web authentication, page doesn't load completely

**Expected behavior**

GUI must work.

**Screenshots**

![image](https://github.com/utmstack/UTMStack/assets/68773119/b59ebc4c-d7a0-464b-820b-9eefde24de4b)

**Environment**

 - OS: Ubuntu 22.04
 - Browser Firefox, Chrome
 - Version UTMStack v10.5.1-202406182313

**Comment by securemeit** [comment]
For your information, gui has come back today.

**Comment by c3s4rfred** [comment]
Hi, @secureme71 in some instances with high logs generation, we detect and fix a bug, please upgrade to the latest version and tell us if the issue is solved.

Best regards

**Comment by securemeit** [comment]
Yes it is fixed.
